scripts/insert-all-ranks.py

```diff
@@ -29,9 +29,5 @@ def run():
                 rank_class=row['class']
         )
         except Exception as e:
-            print("Error with: " + row['name'])
+            print(f"""💥row: ({row}) """)
```

Comment:
fyi, docstrings """ aren't needed for f strings. A single quotation is fine `print(f"💥row: ({row})")`